sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -59,11 +59,14 @@ static void _stale_nc(kernel_pid_t iface, ipv6_addr_t *ipaddr, uint8_t *l2addr,
             gnrc_ipv6_netif_t *ipv6_iface = gnrc_ipv6_netif_get(iface);
             if ((ipv6_iface == NULL) || (ipv6_iface->flags & GNRC_IPV6_NETIF_FLAGS_ROUTER)) {
```

Comment:
Well it would be consistent with the rest of the code then (: Looks like there are several other places in ndp where the result of `gnrc_ipv6_netif_get()` is not properly checked. I wonder why he static analyzer does not complain about this.